.github/workflows/release.yml

```diff
@@ -0,0 +1,85 @@
+name: Build Release
+on:
+  push:
+    tags:
+      - v*
+jobs:
+  create-release:
+    runs-on: 'ubuntu-20.04'
+    outputs:
+      upload_url: ${{ steps.create-release.outputs.upload_url }}
+    steps:
+      - name: Create Release
+        id: create-release
+        uses: ncipollo/release-action@v1
+        with:
+          allowUpdates: true
+          omitBody: true
+          prerelease: true
+          draft: true
+          token: ${{ secrets.ACCESS_TOKEN }}
+  build:
+    runs-on: ${{ matrix.target.os }}
+    needs: create-release
+    strategy:
+      fail-fast: false
+      matrix:
+        target:
+          - os: ubuntu-20.04
+            triple: x86_64-unknown-linux-gnu
+          - os: macos-11
+            triple: x86_64-apple-darwin
+          - os: macos-11
+            triple: aarch64-apple-darwin
+          - os: windows-2019
+            triple: x86_64-pc-windows-msvc
+        features: ['static-link', 'static-link,nozlib']
```

Comment:
I believe dynamic linking with the system libraries should be the default option, the one that is going to be published to crates.io
additionally we can have a release build with assimp compiled and statically linked :)

Reply:
This step compiles the static link library, only the static link feature is opened here